src/classes/SftpSessionHandler.ts

```diff
@@ -126,9 +149,23 @@ export class SftpSessionHandler {
         length,
       },
     );
-    const file = this.openFiles.get(handle.toString());
-    if (!file) {
-      logger.info('There is no open file associated with this handle', { reqId, handle });
+    const serverResource = this.activeHandles.get(handle.toString());
```

Comment:
These checks are repeated in a bunch of places.  Since most of the repetition is in the form of logs and response handling, I'm not sure if it's possible to DRY it out without also inappropriately injecting "handler" logic farther down the stack.  Still I wanted to flag it for consideration.